.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "mplexporter"]
 	path = mplexporter
-	url = https://github.com/mpld3/mplexporter.git
+	url = https://github.com/genomematt/mplexporter.git
```

Comment:
It should be able to see this ok, but it will need to be undone.
I had issues with submodule so I did the changes directly in a fork of mplexporter then pulled them in.

mplexporter also failed with a similar error and its not using a submodule.  I would check out that simpler case first.

> On 23 Mar 2016, at 12:34 AM, Abraham Flaxman <notifications@github.com> wrote:
> 
> In .gitmodules <https://github.com/jakevdp/mpld3/pull/350#discussion_r56985774>:
> 
> > @@ -1,3 +1,3 @@
> >  [submodule "mplexporter"]
> >  	path = mplexporter
> > -	url = https://github.com/mpld3/mplexporter.git
> > +	url = https://github.com/genomematt/mplexporter.git
> This could be the reason TravisCI failed... can you try undo-ing this change?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/jakevdp/mpld3/pull/350/files/97d8478e21d60ba02d9b669b15650fdee984876b#r56985774>


______________________________________________________________________

The information in this email is confidential and intended solely for the addressee.
You must not disclose, forward, print or use it without the permission of the sender.
______________________________________________________________________